mappings/net/minecraft/util/PathUtil.mapping

```diff
@@ -41,11 +41,26 @@ CLASS net/minecraft/class_4239 net/minecraft/util/FileNameUtil
 		COMMENT to {@code /}}
 		ARG 0 path
 	METHOD method_46344 getPath (Ljava/nio/file/Path;Ljava/util/List;)Ljava/nio/file/Path;
+		COMMENT {@return {@code paths} resolved as a path from {@code root}}
+		COMMENT
+		COMMENT <p>If {@code paths} is empty, this returns {@code root}.
 		ARG 0 root
 		ARG 1 paths
-	METHOD method_46345 isPathValid ([Ljava/lang/String;)V
+	METHOD method_46345 validatePath ([Ljava/lang/String;)V
+		COMMENT Validates that {@code paths} is not empty and does not contain invalid segments
+		COMMENT (such as {@code .}, {@code ..}, or otherwise {@linkplain #isFileNameValid invalid names}).
+		COMMENT
+		COMMENT @throws IllegalArgumentException when the {@code paths} is invalid
```

Comment:
```suggestion
		COMMENT @throws IllegalArgumentException when the {@code paths} are invalid
```
"the paths is" sounds a bit weird, I'd say either "`paths` is" (like in the first sentence) or "the `paths` are"